runtime/near-vm-logic/src/dependencies.rs

```diff
@@ -142,6 +142,7 @@ pub trait External {
     /// ```
     fn storage_has_key(&mut self, key: &[u8]) -> Result<bool>;
 
+    /*
```

Comment:
Do you plan to remove this code that you commented out?

Reply:
Yep. Might be leftover